apps/backend/src/handlers/markFormAsRead.ts

```diff
@@ -0,0 +1,44 @@
+import { APIGatewayEvent } from 'aws-lambda';
+import { createTableIfNotExists } from '../db/createTable.js';
+import { markFormAsRead } from '../db/utils.js';
+import { createResponse } from '../utils/createResponse.js';
+import { validateMethodType } from '../utils/validateMethodType.js';
+import { validateRequestBody } from '../utils/validateRequestBody.js';
+import { formSchema } from '../schema/schema.js';
+/**
+ * An HTTP post method to add one form to the QLDB table.
+ */
+export const markAsReadHandler = async (event: APIGatewayEvent) => {
+  let response = validateMethodType(
+    event.httpMethod,
+    'PATCH',
+    'markAsRead'
+  );
+  if (response) {
+    return response;
+  }
+
+  // All log statements are written to CloudWatch
+  console.info('received:', event);
+
+  const id = event.pathParameters!.id!;
+  const JSONBody = JSON.parse(event.body!);
+
+  const isValidBody = validateRequestBody(JSONBody, formSchema);
```

Comment:
I see that this is also a problem in the `patchFormHandler` function, but this is named incorrectly right? It should be isInvalidBody?

Reply:
Changed this to be "isInvalidBody" in my most recent commit.